test/client/fixtures/sandbox/node/window-test.js

```diff
@@ -375,3 +375,15 @@ test('the constructor field of a function should return a wrapped Function objec
     Function.prototype.toString = nativeToString;
     /*eslint-enable no-extend-native*/
 });
+
+test('getPropertyValue and setProperty methods of css object should be overridden (GH-1212)', function () {
```

Comment:
Also check `style.removeProperty`.
https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/removeProperty

It should returns non-proxied value.